storage/aptosdb/src/metrics.rs

```diff
@@ -59,10 +59,17 @@ pub static APTOS_STORAGE_LATEST_ACCOUNT_COUNT: Lazy<IntGauge> = Lazy::new(|| {
     .unwrap()
 });
 
-pub static APTOS_STORAGE_PRUNE_WINDOW: Lazy<IntGauge> = Lazy::new(|| {
+pub static STORAGE_PRUNE_WINDOW: Lazy<IntGauge> = Lazy::new(|| {
```

Comment:
I'd even go as far as `PRUNE_WINDOW`

Reply:
Good point, renamed all relevant counters to remove "APTOS_STORAGE" prefix.